### PR TITLE
UmbraJs and frontend support for Base network

### PR DESCRIPTION
**Pull request by jferas**
Base network support

### CODE REVIEW

File: umbra-js/src/classes/Umbra.ts [hunk 1]
```diff
@@ -56,6 +57,7 @@ const chainConfigs: Record<number, ChainConfig> = {
   100: { chainId: 100, umbraAddress, batchSendAddress, startBlock: 28237950, subgraphUrl: subgraphs[100] }, // Gnosis Chain
   137: { chainId: 137, umbraAddress, batchSendAddress, startBlock: 20717318, subgraphUrl: subgraphs[137] }, // Polygon
   1337: { chainId: 1337, umbraAddress, batchSendAddress, startBlock: 8505089, subgraphUrl: false }, // Local
+  8453: { chainId: 8453, umbraAddress, batchSendAddress, startBlock: 0, subgraphUrl: subgraphs[8453] }, // Base
```
Comment (alexkeating) [inline umbra-js/src/classes/Umbra.ts:60]: Should start block be when Umbra was deployed on Base? @garyghayrat

**Review by alexkeating — APPROVED** [review]
Looks good, the only thing I am wondering is if start block should be something other than 0

File: umbra-js/src/classes/Umbra.ts (hunk 1 above)
Comment (garyghayrat) [reply umbra-js/src/classes/Umbra.ts:60]: yes!

Comment (jferas) [reply umbra-js/src/classes/Umbra.ts]: Thanks guys!  Done: https://github.com/ScopeLift/umbra-protocol/pull/643/commits/a368b4a7ab463e43ae5e4feb7b7d86a592500f96

File: umbra-js/src/classes/Umbra.ts [hunk 2]
```diff
@@ -492,6 +495,7 @@ export class Umbra {
     const errMsg = (network: string) => `Cannot fetch Announcements from logs on ${network}, please try again later`;
     if (this.chainConfig.chainId === 10) throw new Error(errMsg('Optimism'));
     if (this.chainConfig.chainId === 137) throw new Error(errMsg('Polygon'));
+    if (this.chainConfig.chainId === 10) throw new Error(errMsg('Base'));
```
Comment (garyghayrat) [inline umbra-js/src/classes/Umbra.ts:498]:
```suggestion
    if (this.chainConfig.chainId === 8453) throw new Error(errMsg('Base'));
```

**Review by garyghayrat — APPROVED** [review]
Working great, thanks john!

File: umbra-js/src/classes/Umbra.ts (hunk 2 above)
Comment (jferas) [reply umbra-js/src/classes/Umbra.ts]: Good catch! thanks! https://github.com/ScopeLift/umbra-protocol/pull/643/commits/f9e087ec37e7e10884650bbd0b41b1ed980852e3

**Pull request merged**